app/functional_tests_fixtures/__init__.py

```diff
@@ -279,7 +280,8 @@ def _create_organiation(org_name="Functional Tests Org"):
 
         dao_create_organisation(org)
 
-    dao_update_organisation(org.id, can_approve_own_go_live_requests=True)
+    org.set_permissions_list([CAN_ASK_TO_JOIN_SERVICE])
+    dao_update_organisation(org.id, domains=["digital5.cabinet-office.gov.uk"], can_approve_own_go_live_requests=True)
```

Comment:
we define the email domain already on line 93 of this file - can we pass that email domain in to this function?

it'll avoid typos like this one :)